src/app/onboarding/views/OnboardingDataplanesView.vue

```diff
@@ -9,114 +9,103 @@
-              :key="item"
-            >
-              <OnboardingHeading :data-testid="`state-${item}`">
-                <template #title>
-                  {{ t(`onboarding.routes.dataplanes-overview.header.${item}.title`) }}
-                </template>
+                  <template
+                    #description
+                  >
+                    <p>{{ t(`onboarding.routes.dataplanes-overview.header.${item}.description`) }}</p>
+                  </template>
+                </OnboardingHeading>
+              </template>
+            </template>
 
+            <template #content>
+              <DataLoader
+                :data="[data]"
+                :errors="[error]"
+              >
                 <template
-                  #description
+                  #connecting
                 >
-                  <p>{{ t(`onboarding.routes.dataplanes-overview.header.${item}.description`) }}</p>
+                  <div
+                    class="status-loading-box mb-4"
+                  >
+                    <LoadingBox />
+                  </div>
                 </template>
-              </OnboardingHeading>
-            </template>
-          </template>
-
-          <template #content>
-            <div
-              v-if="dataplanes.length === 0"
-              class="status-loading-box mb-4"
-            >
-              <LoadingBox />
-            </div>
-
-            <div v-else>
-              <p class="mb-4">
-                <b>Found {{ dataplanes.length }} DPPs:</b>
-              </p>
 
-              <KTable
-                class="mb-4"
-                data-testid="dataplanes-table"
-                :fetcher-cache-key="String(cacheKey)"
-                :fetcher="() => ({
-                  data: dataplanes,
-                  total: dataplanes?.length,
-                })"
-                :headers="[
-                  { label: 'Mesh', key: 'mesh' },
-                  { label: 'Name', key: 'name' },
-                  { label: 'Status', key: 'status' },
-                ]"
-                disable-pagination
-              >
-                <template #status="{ row }">
-                  <StatusBadge :status="row.status" />
-                </template>
-              </KTable>
-            </div>
-          </template>
+                <div v-if="data">
+                  <p>
+                    <strong>
+                      Found {{ data.items.length }} DPPs:
+                    </strong>
+                  </p>
+                  <KTable
+                    class="mb-4"
+                    data-testid="dataplanes-table"
+                    :fetcher-cache-key="JSON.stringify(data)"
+                    :fetcher="((data) => {
+                      return () => {
+                        return {
+                          data: data.items,
+                          total: data.items.length,
+                        }
+                      }})(data)"
```

Comment:
I noticed that `KTable::fetcher` isn't reactive, i.e. if i change the value of `:fetcher` `KTable` will still use the initial one, which I guess is probably a bug. Not totally sure i might file an issue. Wdyt? Should this be considered a bug?

---

I left in the function factory incase it ever gets fixed